_data/contributors.yml

```diff
@@ -22,17 +25,35 @@
     - Just the beginning
     - Configuration Demystified
     - Ecto query composition
+    - 'Umbrellas: only when it rains?'
```

Comment:
It's not liking this for some reason, going to work with @burden to sort it out.

Reply:
Might be the quotes

Reply:
It doesn't like it without them either 😞